var/spack/repos/builtin/packages/silo/package.py

```diff
@@ -44,8 +44,7 @@ class Silo(AutotoolsPackage):
     depends_on('automake', type='build', when='+shared')
     depends_on('libtool', type='build', when='+shared')
     depends_on('mpi', when='+mpi')
-    depends_on('hdf5', when='+hdf5')
-    depends_on('hdf5 api=v110', when='@:4.10 +hdf5 ^hdf5@1.12:')
+    depends_on('hdf5@1.8:', when='+hdf5')
```

Comment:
This should be constrained to `@1.10` right? hdf5@1.8 doesn't support the macros for the 1.10 API.